crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -91,11 +89,9 @@ async fn app_validation_workflow_inner(
     let sorted_ops = validation_query::get_ops_to_app_validate(&db).await?;
```

Comment:
These ops are sorted sorted according to the database query. You might actually want to sort them there. Doesn't need to be in this PR